app/components/dossiers/batch_operation_component.rb

```diff
@@ -7,17 +7,25 @@ def initialize(statut:, procedure:)
   end
 
   def render?
-    ['a-suivre', 'traites', 'suivis', 'archives', 'supprimes_recemment'].include?(@statut)
+    ['a-suivre', 'traites', 'suivis', 'archives', 'supprimes_recemment', 'expirant'].include?(@statut)
   end
 
   def operations_for_dossier(dossier)
     case dossier.state
+    when Dossier.states.fetch(:brouillon)
+      [BatchOperation.operations.fetch(:repousser_expiration)]
```

Comment:
pas necessaire, les instructeurs n'ont pas accès aux dossiers en brouillon

Reply:
jai mis brouillon a cause de ce bout de code qui prend en compte les brouillons

`


  def expiration_date_reference
    if brouillon?
      created_at
    elsif en_construction?
      en_construction_at
    elsif termine?
      processed_at
    else
      fail "expiration_date_reference should not be called in state #{self.state}"
    end
  end

  def expiration_date_with_extension
    expiration_date_reference + conservation_extension + procedure.duree_conservation_dossiers_dans_ds.months
  end

  def expiration_notification_date
    expiration_date_with_extension - Expired::REMAINING_WEEKS_BEFORE_EXPIRATION.weeks
  end

  def close_to_expiration?
    return false if en_instruction?
    expiration_notification_date < Time.zone.now
  end
`

tu me confirmes donc bien que je ne dois pas prendre en compte les brouillon ?